experimental/PHP/README.md

```diff
@@ -0,0 +1,45 @@
+# PHP On IIS
+
+## What is PHP
+
+PHP (PHP: Hypertext Processor) is a server side scripting language as well as web framework that can be used to develop applications ranging from simple to complex.
+
+## Getting Started
+
+To add a PHP application into the container, first put the application content into a folder named _content_ and then append the following commands to the dockerfile located within this directory
+
+```
+# PHP dockerfile
+# The content of the docker file will be here
+# Add your content below
+
+RUN mkdir C:\site
+
+RUN powershell -NoProfile -Command \
+    Import-module IISAdministration; \
+    New-IISSite -Name "Site" -PhysicalPath C:\site -BindingInformation "*:8000:"

```

Comment:
New-IISSite will break ServiceMonitor behavior. Use Deafult Web Site

Reply:
@shirhatti 

I just used the same pattern present on the main page README. Is that also broken?